src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs

```diff
@@ -1,3 +1,5 @@
+using Nancy.Configuration;

```

Comment:
No, but I normally don't contribute to these satellite repos, and they don't have one. That's the only reason I get things right for Nancy core. Life's too short to manually import (and think about) namespaces.